src/main.ts

```diff
@@ -334,30 +382,17 @@ export default class NewZettel extends Plugin {
             : selectionPos.anchor.line < selectionPos.head.line; // else they are not on the same line and just check if anchor is before head
 
         // if anchorCorrect use as is, else switch
-        const virtualAnchor = anchorCorrect
-          ? selectionPos.anchor
-          : selectionPos.head;
-        const virtualHead = anchorCorrect
-          ? selectionPos.head
-          : selectionPos.anchor;
-        editor!.replaceRange(
-          " ".repeat(spaceBefore) + newLink(title) + " ".repeat(spaceAfter),
-          virtualAnchor,
-          virtualHead
-        );
-        this.makeNote(nextPath(title), title, fileLink, true, openNewFile);
+        const virtualAnchor = anchorCorrect ? selectionPos.anchor : selectionPos.head;
+        const virtualHead = anchorCorrect ? selectionPos.head : selectionPos.anchor;
```

Comment:
I like the compression of these 2 lines good job!

Reply:
I think that was my formatter at the time. I tried no to mess with the formatting of the original code, but it was near a part that was changed by me, so I probably committed the formatting as well. But if it's okay, then no problem, I guess. :)